code/modules/vehicles/speedbike.dm

```diff
@@ -520,7 +520,14 @@
 
 /obj/vehicle/space/speedbike/speedwagon/buckle_mob(mob/living/M, force = 0, check_loc = 1)
  	. = ..()
-		riding_datum = new/datum/riding/space/speedbike/speedwagon
+    	riding_datum = new/datum/riding/space/speedwagon
```

Comment:
if you're using tortoisegit the best way to deal with these is to pull expecting a failure, close the pull tab and YES to show diff then right click the BLOOD RED coloured file line in the list of diffs and use resolve using "mine" or "Theirs" depending

if not just copy paste raw into the file ctrl d then ctrl c to raw ctrl v to DM